index.d.ts

```diff
@@ -0,0 +1,22 @@
+declare module '@discoveryjs/json-ext' {
+    import { Readable } from 'stream';
+
+    export function parseChunked<T>(stream: Readable | Generator<any, void, unknown>): Promise<T>;
+
+    export function stringifyStream(value: any, replacer?: ((this: any, key: string, value: any) => any) | undefined, space?: string | number | undefined): Readable;
+
+    export function stringifyInfo(
+        value: any,
+        replacer?: ((this: any, key: string, value: any) => any) | undefined,
```

Comment:
if a field is optional `foo?:` the there is no need `| undefined`
here and below

Reply:
Yes, I just only copy typing from `JSON.stringify`. Fixed